backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/RunVmCommandBase.java

```diff
@@ -409,7 +409,8 @@ protected void addNumaPinningForDedicated(Guid vdsId) {
     }
 
     protected void setDedicatedCpus(VdsManager vdsManager) {
-        if (getVm().getCpuPinningPolicy() != CpuPinningPolicy.DEDICATED) {
+        if (getVm().getCpuPinningPolicy() != CpuPinningPolicy.DEDICATED &&
```

Comment:
What about adding a new method isExclusive() to CpuPinningPolicy that would return true for DEDICATED and ISOLATE_THREADS?